acceptance/tests/modules/build/build_agent.rb

```diff
@@ -1,5 +1,8 @@
 test_name "puppet module build (agent)"
 
+tag 'audit:medium',
+    'audit:acceptance'
```

Comment:
I think my concern here was to ensure that the build tooling was properly supported on the agent OSs (as module development platforms). If that is not a risk, then this test should simply be deleted, since the build command is tested elsewhere.